lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -502,16 +503,59 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid,
 }
 
 static void
-blockdev_nvme_timeout_cb(void *cb_arg, struct spdk_nvme_ctrlr *ctrlr,
-			 struct spdk_nvme_qpair *qpair, uint16_t cid)
+abort_cb(void *arg, const struct spdk_nvme_cpl *cpl)
+{
+	/*
+	 * abort completion callback function. Currently, there is nothing to do
+	 * struct spdk_nvme_ctrlr *ctrlr = arg;
+	 */
+}
+
+static void
+blockdev_nvme_timeout_cb(struct spdk_nvme_ctrlr *ctrlr,
+			 struct spdk_nvme_qpair *qpair,
+			 void *cb_arg,
```

Comment:
Ben, resetting the controller if the abort fails may not be desired here because when i tested the abort command, i found most of the time when abort is a issued,  "the command to abort may have already completed" and you get from the admin completion queue DWORD 0 bit 0 is set to 1 for the abort command. The status code and status code type are set to 0 because the abort command executed successfully although mission was not accomplished. 
This is abort command behavior but controller itself is fine, therefore, resetting the controller may not be desired.